test/tests/functional/pbs_pbsnodes_output_trimmed.py

```diff
@@ -1,6 +1,6 @@
 # coding: utf-8
 
-# Copyright (C) 1994-2019 Altair Engineering, Inc.
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
```

Comment:
2019

Reply:
@agrawalravi90 I've pushed the changes for the same. Kindly review.